packages/frontend/src/pages/Inbox/OldInboxNotification.tsx

```diff
@@ -0,0 +1,43 @@
+import { NotificationItem } from '@altinn/altinn-components';
+import { BellIcon } from '@navikt/aksel-icons';
+import type { Altinn2Message } from 'bff-types-generated';
+import { useTranslation } from 'react-i18next';
+import { useAltinn2Messages } from '../../api/hooks/useAltinn2Messages.tsx';
+
+export const OldInboxNotification = () => {
```

Comment:
`OldInboxNotification` suggests that there is also a “new” version of this component, which isn’t the case. It also doesn’t describe the actual purpose of the component. Some suggestions:

- UnmigratedMessageNotification
- Altinn2MessageNotification

Probably better to make the intent clearer?